Chummer/data/vehicles.xml

```diff
@@ -10203,6 +10203,236 @@
+        <name>Winch (Basic)</name>
+        <name>Searchlight</name>
+        <name>Extreme Environment Modification</name>
+        <name rating="1">Life Support</name>
+        <name select="Firefighting Cannon">Special Equipment</name>
+        <name select="Dozer Blade">Special Equipment</name>
+        <name select="Water Jet Cutter">Special Equipment</name>
+        <name select="Rescue-Equipment">Special Equipment</name>
+        <name rating="6">Special Armor Modification (Fire)</name>
+      </mods>
+      <seats>6</seats>
+      <weaponmounts>
+        <weaponmount>
+          <control>Remote</control>
+          <flexibility>Turret</flexibility>
+          <size>Heavy</size>
+          <visibility>External</visibility>
+        </weaponmount>
+      </weaponmounts>
+      <weapons>
+        <weapon>
+          <name>Krupp Munitions 3E Firefighting Cannon</name>
+        </weapon>
+      </weapons>
+    </vehicle>
+    <vehicle>
+      <id>1af96573-ee09-4bc9-8598-75f62bccfebc</id>
+      <name>Eurocar Escape DeKW</name>
+      <source>SOTG</source>
+      <page>16</page>
+      <accel>2</accel>
+      <armor>14</armor>
+      <avail>14</avail>
+      <body>18</body>
+      <category>Trucks</category>
+      <cost>225000</cost>
+      <handling>3/4</handling>
+      <pilot>3</pilot>
+      <sensor>2</sensor>
+      <speed>3</speed>
+      <gears>
+        <gear>
+          <name>Sensor Array</name>
+          <rating>1</rating>
+          <maxrating>6</maxrating>
+        </gear>
+      </gears>
+      <mods>
+        <name rating="2">Anti-Theft System</name>
+        <name>Standard Drone Rack (Small)</name>
+        <name>Off-Road Suspension</name>
+        <name>Amenities (Low)</name>
+        <name>Multifuel Engine</name>
+        <name>Satellite Link</name>
+        <name>Winch (Basic)</name>
+        <name>Searchlight</name>
+        <name rating="2">Life Support</name>
+        <name select="Firefighting Cannon">Special Equipment</name>
+        <name select="Dozer Blade">Special Equipment</name>
+        <name select="ABC-Equipment">Special Equipment</name>
+        <name rating="6">Special Armor Modification (Radiation)</name>
+        <name rating="6">Special Armor Modification (Chemical)</name>
+      </mods>
+      <seats>6</seats>
+      <weaponmounts>
+        <weaponmount>
+          <control>Remote</control>
+          <flexibility>Turret</flexibility>
+          <size>Heavy</size>
+          <visibility>External</visibility>
+        </weaponmount>
+      </weaponmounts>
+      <weapons>
+        <weapon>
+          <name>Krupp Munitions 3E Firefighting Cannon</name>
+        </weapon>
+      </weapons>
+    </vehicle>
+    <vehicle>
+      <id>7d708ed4-02ce-4047-91bb-6bce6bee03aa</id>
+      <name>THW MzKW C</name>
+      <source>SOTG</source>
+      <page>16</page>
+      <accel>2</accel>
+      <armor>10</armor>
+      <avail>8</avail>
+      <body>14</body>
+      <category>Cars</category>
```

Comment:
Why is this MzKW part of Cars when the other one is part of Trucks?

Reply:
Because they are different vehicles, the MzKW D is based on the Eurocar Escape as a base vehicle, the MzKW C is based on the BMW X Infinity as a base vehicle